src/configuration/QuerySettings/types.ts

```diff
@@ -7,6 +7,9 @@ export interface QuerySettings {
   format?: string;
   contextParameters?: QueryContextParameter[];
   hideEmptyColumns?: boolean;
+  logColumnTime?: string;
```

Comment:
This could have been a log object, so its interface/content would have been under responsibility of log settings component (not a blocker IMHO, let's move ahead that way but if you feel the change: feel free :))